react/components/MiniCartContent.js

```diff
@@ -185,7 +201,7 @@ class MiniCartContent extends Component {
             <Fragment key={item.id}>
               <div className="relative flex">
                 <div className="fr absolute bottom-0 right-0">
-                  <Button icon variation="tertiary" onClick={e => this.onRemoveItem(item.id)}>
+                  <Button icon variation="tertiary" disabled={isUpdating} onClick={e => this.onRemoveItem(item.id)}>
```

Comment:
You don't have to receive the event if you dont use it. And I think the name of the method should be `handleRemoveItem`